### PR TITLE
Add add method to companylist and handle input mismatch exception

### PR DESCRIPTION
**Pull request by de-yi**

### CODE REVIEW

File: src/main/java/CompanyList.java [hunk 1]
```diff
@@ -5,4 +5,14 @@
 public class CompanyList {
 
     ArrayList<Company> companies = new ArrayList<>();
+
+    public boolean add(String companyName, int contactNumber, String contactEmail){
```
Comment (kishore-a00) [inline src/main/java/CompanyList.java:9]: Perhaps add a space between the closing parenthesis and the opening curly braces.

File: src/main/java/CompanyList.java [hunk 2]
```diff
@@ -5,4 +5,14 @@
...
+        try{
+            Company newCompany = new Company(companyName, contactNumber, contactEmail);
+            companies.add(newCompany);
+            return true;
+        } catch(seedu.duke.InputMismatchException e){
```
Comment (kishore-a00) [inline src/main/java/CompanyList.java:14]: Perhaps add a space between the closing parenthesis and the opening curly braces.

File: src/main/java/eventus/company/Company.java [hunk 3]
```diff
@@ -15,7 +15,7 @@ public String getCompanyName(){
         return companyName;
     }
 
-    public String getContactNumber(){
+    public int getContactNumber(){
```
Comment (kishore-a00) [inline src/main/java/eventus/company/Company.java:18]: Perhaps add a space between the closing parenthesis and the opening curly braces.

File: src/main/java/seedu/duke/InputMismatchException.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package seedu.duke;
+
+public class InputMismatchException extends Exception{
```
Comment (kishore-a00) [inline src/main/java/seedu/duke/InputMismatchException.java:3]: Perhaps add a space right before the opening curly braces.

**Review by kishore-a00 — COMMENTED** [review]
Looks good to me, just a few minor formatting issues.

**Comment by wuzhzn** [comment]
Looks good to merge

**Pull request merged**